cli/utils/constants.py

```diff
@@ -101,9 +101,7 @@
 
     # Network configuration
     network = "{ctx.network}"
```

Comment:
Needs a `use_vpc = "{ctx.use_vpc}"`.

You will also need to change the `testdata/dummy_stage_v3.py` stage file accordingly.